code/game/objects/structures/safe.dm

```diff
@@ -286,4 +286,24 @@ FLOOR SAFES
 	new /obj/random/highvalue(src)
 	new /obj/random/highvalue(src)
 	new /obj/random/highvalue(src)
-	new /obj/random/highvalue(src)
\ No newline at end of file
+	new /obj/random/highvalue(src)
+
```

Comment:
stray newline.

Reply:
New line is intentional to provide a gap between the corporate safe and credit safe entries, unless I'm missing something about it.